src/components/FlawCreateForm.vue

```diff
@@ -83,11 +93,11 @@ const getUnembargoDateTime = () => {
           <div class="col-6 text-end">
             <div class="row">
               <div class="col-6">
-                <div class="input-group mb-2" :class="{major: modelValue.is_major_incident}">
+                <div class="input-group mb-2" :class="{major: modelValue.is_major_incident, 'bg-danger': modelValue.is_major_incident}">
```

Comment:
Why add a duplicate class?
Also update the value to use the new field instead of the deprecated field